X16 Reference - 11 -  Working with CMDR-DOS.md

```diff
@@ -0,0 +1,368 @@
+# Chapter 11: Working With CMDR-DOS
+This manual describes Commodore DOS on FAT32, aka CMDR-DOS. 
+
+## CMDR-DOS
+
+Commander-16 duplicates and extends the programming interface used by Commodore's line of disk drives, including the 
+famous (or infamous) VIC-1541. CMDR-DOS uses the industry-standard FAT-32 format. Partitions can be 32MB up to 
+(in theory) 2TB and supports CMD-style partitions, subdirectories, timestamps and filenames up to 255 characters. 
+It is the DOS built into the [Commander X16](https://www.commanderx16.com).
+
+There are three basic interfaces for CMDR-DOS: the binary interface (SAVE,LOAD), the data file interface (OPEN, 
```

Comment:
nitpick about formatting, perhaps a space after the comma

Reply:
agreed. made "LOAD, SAVE, etc." with all the B/V variants and VERIFY commands.